bitshares/bitshares-community-ui
# 67 1st release integration

#### What's this PR do?
Adds keyfile login functionality + integrates, fixes and stylized Main Dashboard with Account and Transactions widgets
#### Where should the reviewer start?
src/modules/app.js
src/views/Dashboard.vue
#### How should this be manually tested?
Try logging in with .bin file and check out your transactions history and assets
#### Any background context you want to provide?
Don't forget to pull laters `vuex-bitshares` dev branch
#### What are the relevant tickets?
closes #66 
closes #67 
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/4411295/46909696-4dcd7580-cf3f-11e8-86a8-710c4b898d17.png)

#### Questions:
n/a

#### URL
http://staging-ui.trusty.apasia.tech:8080/66-key-file-integration
